js/loader.js

```diff
@@ -26,7 +26,7 @@ var sceneChecked;
 var sceneLocation = "../../../Scenes/";
 
 // Babylon
-var engine = new BABYLON.Engine(canvas, true);
+var engine = new BABYLON.Engine(canvas, true, { preserveDrawingBuffer: true });
```

Comment:
Are you aware of any downsides of me doing this for all scenes?

Reply:
No pb this is fine